x-pack/plugins/apm/common/utils/formatters/duration.ts

```diff
@@ -186,7 +186,7 @@ export function asDuration(
  * `asDuration`, but this is used in places like tables where we always want
  * the same units.
  */
-export function asMillisecondDuration(time: number) {
+export function asMillisecondDuration(time: Maybe<number>) {
```

Comment:
nit: while you are at it: can you rename this?

```suggestion
export function asMillisecondDuration(value: Maybe<number>) {
```